app/javascript/styles/components/Calendar.scss

```diff
@@ -89,3 +104,58 @@
     text-align: center;
   }
 }
+
+#jobs-year {
+  color: $color_font-medium;
+  font-family: $font_family-primary;
+  font-weight: bolder;
+  font-size: 60px;
+  margin-bottom: 1.5rem;
+}
+
+.jobs {
+  margin-right: 5rem;
+
+  text {
```

Comment:
Would this work with a class selector instead?

Reply:
Yes it would. Turns out not needed, now that I've refactored this into classes and doing it all through D3.